src/pykeen/models/unimodal/mure.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+
+"""Implementation of MuRE."""
+from typing import Any, ClassVar, Mapping
+
+import torch
+
+from ..nbase import ERModel
+from ...constants import DEFAULT_EMBEDDING_HPO_EMBEDDING_DIM_RANGE
+from ...nn import EmbeddingSpecification
+from ...nn.init import xavier_normal_
+from ...nn.modules import MuREInteraction
+from ...typing import Hint, Initializer
+
+__all__ = [
+    'MuRE',
+]
+
+
+class MuRE(ERModel):
+    r"""An implementation of MuRE from [balazevic2019b]_.
+
+    ---
+    citation:
+        author: Balažević
+        year: 2019
+        link: https://arxiv.org/abs/1905.09791
+    """
+
+    #: The default strategy for optimizing the model's hyper-parameters
+    hpo_default: ClassVar[Mapping[str, Any]] = dict(
+        embedding_dim=DEFAULT_EMBEDDING_HPO_EMBEDDING_DIM_RANGE,
+        p=dict(type=int, low=1, high=2),
+    )
+
+    def __init__(
+        self,
+        *,
+        embedding_dim: int = 200,
+        p: int = 2,
+        power_norm: bool = True,
+        entity_initializer: Hint[Initializer] = xavier_normal_,
```

Comment:
these are currently unused. should we change the names, add more, and make them all configurable?

Reply:
Yes, I was think so. Would this pose any problem with the pipeline?

Reply:
nope!

Reply:
done in 7edc29a